workflow-dev/tpl/deis-objectstorage-secret.yaml

```diff
@@ -1,4 +1,4 @@
-#helm:generate helm template -o $HELM_GENERATE_DIR/manifests/deis-objectstorage-secret.yaml -d $HELM_GENERATE_DIR/tpl/generate_params.toml $HELM_GENERATE_DIR/tpl/deis-objectstorage-secret.yaml
+#helm:generate helmc template -o $HELM_GENERATE_DIR/manifests/deis-objectstorage-secret.yaml -d $HELM_GENERATE_DIR/tpl/generate_params.toml $HELM_GENERATE_DIR/tpl/deis-objectstorage-secret.yaml
```

Comment:
I think this header should continue to reference `helm`, not `helmc`. That's what we have in the workflow-beta4 chart and I believe Helm Classic intends to keep it that way for backward compatibility. ping @krancour.